docs/cni-plugin.md

```diff
@@ -35,16 +35,33 @@ Another example with a trunk port and jumbo frames:
 }
 ```
 
+Another example with QinQ configured:
+
+```json
+{
+    "name": "myqinqnet",
+    "type": "ovs",
+    "bridge": "mynet2",
+    "vlan": 1000,
```

Comment:
Don't forget to add the vlanMode line here.

Reply:
Right, added